packages/example-project/component-library/.prettierignore

```diff
@@ -0,0 +1,2 @@
+# autogenerated files from stencil
+src/components.d.ts
```

Comment:
TODO: This either didn't get picked up or I accidentally recommitted these changes